pkg/models/operator.go

```diff
@@ -141,6 +142,17 @@ const (
 
 	Username = "username"
 	Password = "password"
+
+	Csr                = "csr"
+	ID                 = "id"
+	ClusterID          = "clusterID"
+	ExpiryDate         = "expiryDate"
+	SignedCertificate  = "signedCertificate"
+	ValidPeriod        = "validPeriod"
+	CommonName         = "commonName"
+	Country            = "country"
+	Organization       = "organization"
+	OrganizationalUnit = "organizationalUnit"
```

Comment:
Where do you use these variables? Let's remove these code

Reply:
done